setup.py

```diff
@@ -137,7 +137,8 @@ def run(self):
         'traittypes',
         'traitlets',
         'Pillow',
-        'ipywebrtc'
+        'ipywebrtc',
+        'requests'
```

Comment:
Ok, I leave it up to you, but the less dependencies the better I'd think. And I like that urlretrieve is a oneliner (except for the import).